santibout/sun-synergy
# [EXAMPLE PR] COD-39: Add Redux for login form

Description
added a redux library that saves the user input and password.

Links
https://redux-toolkit.js.org/tutorials/quick-start

How to Test
npm i
npm run dev
Visit `localhost:3000
Enter an email and password
Press submit and verify that the email and password are logged in the JS console
